### PR TITLE
[15.0][mig] purchase_fop_shipping: Migration to 15.0 

### PR DESCRIPTION
**Pull request by Ricardoalso**

### CODE REVIEW

File: purchase_fop_shipping/i18n/purchase_fop_shipping.pot [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* purchase_fop_shipping
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```
Comment (cyrilmanuel) [inline purchase_fop_shipping/i18n/purchase_fop_shipping.pot:7]: to be ask if need to change version by 15

**Review by cyrilmanuel — APPROVED** [review]
not an expert but LGTM

File: purchase_fop_shipping/__manifest__.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Purchase Free-Of-Payment shipping",
-    "version": "14.0.1.1.0",
```
Comment (leemannd) [inline purchase_fop_shipping/__manifest__.py]: This change is usually done into the migration commit

**Review by leemannd — APPROVED** [review]
Thank you

**Review by santostelmo — APPROVED** [review]
LG thank you

File: purchase_fop_shipping/i18n/purchase_fop_shipping.pot (hunk 1 above)
Comment (leemannd) [reply purchase_fop_shipping/i18n/purchase_fop_shipping.pot:7]: AFAIR this should be automatically updated by the ocabot

File: purchase_fop_shipping/models/purchase.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# © 2017 Akretion (http://www.akretion.com)
```
Comment (simahawk) [inline purchase_fop_shipping/models/purchase.py]:
```suggestion
# Copyright 2017 Akretion (http://www.akretion.com)
```
nitpicking, as you know ;)

File: purchase_fop_shipping/models/purchase.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                    record.amount_total,
+                    record.partner_id.fop_shipping,
+                    precision_digits=digit_precision,
+                )
+                >= 0
+            )
+
+    def button_approve(self, force=False):
+        self._check_fop_shipping()
+        result = super(PurchaseOrder, self).button_approve(force=force)
```
Comment (simahawk) [inline purchase_fop_shipping/models/purchase.py]:
```suggestion
        result = super().button_approve(force=force)
```
same

**Review by simahawk — APPROVED** [review]

File: purchase_fop_shipping/models/purchase.py (hunk 3 above)
Comment (Ricardoalso) [reply purchase_fop_shipping/models/purchase.py]: Done, thanks :smile:

**Comment by simahawk** [comment]
@Ricardoalso please checkout 15.0, run `copier -f update` and open a PR like this https://github.com/OCA/connector/pull/455

See https://github.com/OCA/oca-addons-repo-template/

**Comment by Ricardoalso** [comment]
> @Ricardoalso please checkout 15.0, run `copier -f update` and open a PR like this [OCA/connector#455](https://github.com/OCA/connector/pull/455)
> 
> See https://github.com/OCA/oca-addons-repo-template/

Done here https://github.com/OCA/purchase-workflow/pull/1636

**Pull request merged**